tasks/utilities/submission/task_mercury_file_wrangling.wdl

```diff
@@ -264,6 +264,9 @@ task sm_metadata_wrangling { # the sm stands for supermassive
 
         if update_country == "true":
           genbank_metadata["country"] = genbank_metadata["country"] + ": " + genbank_metadata["state"]
+        
+        # remove state column from genbank
+        genbank_metadata.drop("state", axis=1, inplace=True)
```

Comment:
@sage-wright do we want this nested within the if block above?

Reply:
yes otherwise we'll have a `": "` when there is no state provided

Reply:
No, because I think it will fail if state is included even if the country doesn't have the `: ` addition, but @emily-smith1 might know more